lib/player/http_video_source.js

```diff
@@ -168,6 +171,13 @@ shaka.player.HttpVideoSource.prototype.enableTextTrack = function(enabled) {
 
 
 /** @override */
+shaka.player.HttpVideoSource.prototype.setPlaybackStartTime =
+    function(startTime) {
+  this.playbackStartTime_ = startTime;
```

Comment:
Either this should be a no-op for HttpVideoSource, or this value should be used to set video.currentTime in attach().